### YamlMime:ManagedReference
items:
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException
  id: EntityDataSourceValidationException
  children:
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  langs:
  - csharp
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException
  type: Class
  summary: "Represents errors that occur when validating properties of a dynamic data source."
  syntax:
    content: 'public sealed class EntityDataSourceValidationException : Exception, System.Web.DynamicData.IDynamicValidatorException'
  inheritance:
  - System.Exception
  implements:
  - System.Web.DynamicData.IDynamicValidatorException
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  id: '#ctor'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Creates a new instance of the <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;></xref> class with a system-supplied message that describes the error."
  remarks: "This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as &quot;DefaultMessageDisplayedByParameterlessCtorWriterMustSupply&quot;. This message takes into account the current system culture.       The following table shows the initial property values for an instance of <xref:System.InsufficientMemoryException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The localized error message string.|"
  syntax:
    content: public EntityDataSourceValidationException ();
    parameters: []
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Creates a new instance of the <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;></xref> class with a specified message that describes the error."
  remarks: "The following table shows the initial property values for an instance of <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string specified in `message`.|"
  syntax:
    content: public EntityDataSourceValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Creates a new instance of the <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."
  remarks: "An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.       The following table shows the initial property values for an instance of <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>.      |Property|Value|   |--------------|-----------|   |<xref:System.Exception.InnerException%2A>|`null`.|   |<xref:System.Exception.Message%2A>|The error message string specified in `message`.|"
  syntax:
    content: public EntityDataSourceValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture."
    - id: innerException
      type: System.Exception
      description: "The exception that is the cause of the current exception. If the `innerException` parameter is not <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>, the current exception is raised in a <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref> block that handles the inner exception."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  id: System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  isEii: true
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  type: Property
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "Gets the exceptions that occur when a new or edited data field is validated."
  remarks: "This member is an explicit interface member implementation. It can be used only when the <xref:System.Web.UI.WebControls.EntityDataSourceValidationException> instance is cast to an <xref:System.Web.DynamicData.IDynamicValidatorException> interface."
  syntax:
    content: System.Collections.Generic.IDictionary<string,Exception> System.Web.DynamicData.IDynamicValidatorException.InnerExceptions { get; }
    return:
      type: System.Collections.Generic.IDictionary{System.String,System.Exception}
      description: "A collection that contains all the exceptions that occurred."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
- uid: System.Collections.Generic.IDictionary{System.String,System.Exception}
  parent: System.Collections.Generic
  isExternal: true
  name: IDictionary<String,Exception>
  nameWithType: IDictionary<String,Exception>
  fullName: System.Collections.Generic.IDictionary<System.String,System.Exception>
  spec.csharp:
  - uid: System.Collections.Generic.IDictionary`2
    name: IDictionary
    nameWithType: IDictionary
    fullName: IDictionary<System.String,System.Exception>
  - name: <
    nameWithType: <
    fullName: <
  - uid: System.String
    name: String
    nameWithType: String
    fullName: String
  - name: ','
    nameWithType: ','
    fullName: ','
  - uid: System.Exception
    name: Exception
    nameWithType: Exception
    fullName: Exception
  - name: '>'
    nameWithType: '>'
    fullName: '>'
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
